src/main/java/org/terasology/signalling/components/SignalTimeDelayModifiedComponent.java

```diff
@@ -19,7 +19,8 @@
 import org.terasology.world.block.ForceBlockActive;
 
 /**
- * @author Marcin Sciesinski <marcins78@gmail.com>
+ * Responsible for keeping track of whether a TimeDelay in a Signal has been modified.
+ *
```

Comment:
No need for an extra empty line if there's nothing else in the block below it :-)